.pipelines/azure_pipeline_mergedbranches.yaml

```diff
@@ -236,13 +237,170 @@ jobs:
         az account set -s ${{ variables.subscription }}
         az acr login -n ${{ variables.containerRegistry }}
 
-        docker build --isolation=hyperv --tag ${{ variables.repoImageName }}:$(windowsImageTag)-$(windows2019BaseImageVersion) --build-arg WINDOWS_VERSION=$(windows2019BaseImageVersion) --build-arg IMAGE_TAG=$(windowsTelemetryTag) --build-arg WINDOWS_AMA_URL=$(WINDOWS_AMA_URL) .
+        docker build --isolation=hyperv --tag ${{ variables.repoImageName }}:$(windowsImageTag)-$(windows2019BaseImageVersion)-unsigned --build-arg WINDOWS_VERSION=$(windows2019BaseImageVersion) --build-arg IMAGE_TAG=$(windowsTelemetryTag) --build-arg WINDOWS_AMA_URL=$(WINDOWS_AMA_URL) .
+
+  - task: PowerShell@2
+    displayName: Extract files to sign
+    inputs:
+      targetType: 'inline'
+      script: |
+        echo "Creating docker container..."
+        docker create --name signingContainer ${{ variables.repoImageName }}:$(windowsImageTag)-$(windows2019BaseImageVersion)-unsigned
+
+        echo "Creating fist party directory..."
+        mkdir -p $(Build.ArtifactStagingDirectory)/fpSigning
+        cd $(Build.ArtifactStagingDirectory)/fpSigning
+
+        echo "Extract CertificateGenerator"
+        docker cp signingContainer:C:\opt\amalogswindows\certgenerator\CertificateGenerator.exe .
+        docker cp signingContainer:C:\opt\amalogswindows\certgenerator\CertificateGenerator.dll .
+
+        echo "Extract LivenessProbe"
+        docker cp signingContainer:C:\opt\amalogswindows\scripts\cmd\livenessprobe.exe .
+
+        echo "Extract ps scripts"
+        mkdir -p powershell
```

Comment:
why do we need this since you are copying the binaries from container to current directory?

Reply:
changed